arex-web-api/src/main/java/com/arextest/web/api/service/beans/InterceptorConfiguration.java

```diff
@@ -58,6 +58,9 @@ public void addInterceptors(InterceptorRegistry registry) {
     defaultPatterns.add("/api/report/analyzeCompareResults");
     defaultPatterns.add("/api/report/removeRecordsAndScenes");
     defaultPatterns.add("/api/desensitization/listJar");
+    defaultPatterns.add("/api/system/config/list");
```

Comment:
If storage does not call it, remove this line.

Reply:
in this transformation, schedule will call it